### PR TITLE
docs: add install instructions

### PR DESCRIPTION
**Pull request by y-lakhdar**
add information on how to install package from github package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,50 @@ A Coveo Push API Client in Java
...
-  <groupId>com.coveo</groupId>
-  <artifactId>push-api-client.java</artifactId>
-  <version>2.2.0</version>
-</dependency>
+
+### Step 3: Add Coveo dependency to project
+Using Maven:
+
+
+Add Coveo dependency to your maven project. Instructions on how to do that available in this [URL](https://github.com/coveo/push-api-client.java/packages/1884180).
```
Comment (mrrajamanickam-coveo) [inline README.md:45]: Guessing `1884180` is the pkg id that is not specific to the particular version/release?

File: README.md [hunk 2]
```diff
@@ -4,14 +4,50 @@ A Coveo Push API Client in Java
 
 ## Installation
 
-Using Maven:
+### Step 1: Prerequisites
+The Coveo `push-api-client.java` package is stored on Github packages. You will then need a personal access token (classic) with at least `read:packages` scope to install this dependency.
```
Comment (mrrajamanickam-coveo) [inline README.md:8]:
```suggestion
The Coveo `push-api-client.java` package is stored on Github packages. You will need a personal access token (classic) with at least `read:packages` scope to install this dependency.
```

**Review by mrrajamanickam-coveo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (louis-bompart) [reply README.md:45]: Yep indeed :)

Comment (y-lakhdar) [reply README.md:45]: no. the version is added as a query string

**Pull request merged**